dbcrossbarlib/src/if_exists.rs

```diff
@@ -81,7 +81,7 @@ impl IfExists {
                 open_options.create_new(true);
             }
             IfExists::Overwrite => {
-                open_options.create(true).append(true);
+                open_options.create(true).truncate(true);
```

Comment:
oh that's important.

Reply:
Ironically, this may be the first time I've encountered a "data *non-*loss bug" that I can think of. :-)